Video.js

```diff
@@ -14,8 +14,8 @@ const styles = StyleSheet.create({
   },
 });
 
-const { VideoDecoderProperties } = NativeModules
-export { TextTrackType, FilterType, DRMType, VideoDecoderProperties }
+const { VideoDecoderProperties } = NativeModules;
+export { TextTrackType, FilterType, DRMType, VideoDecoderProperties };
 
 export default class Video extends Component {
 
```

Comment:
changes are not related this this feature, can you revert them please ?

Reply:
@freeboub This is change made by prettier "on save". Do we really need to revert this ? If Yes then should I disable prettier completely ?

Reply:
only to lines, That's ok let's keep it